cli/tests/test_cli.py

```
@@ -0,0 +1,1491 @@
+import pytest
+import json
+import tempfile
+import os
+from pathlib import Path
+from unittest.mock import patch, mock_open, MagicMock
```

Ruff (0.11.9)

`unittest.mock.mock_open` imported but unused

Remove unused import: `unittest.mock.mock_open`

(F401)